### PR TITLE
you can no longer stuff a bajillion live mice into your chef's hat

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

bug bad?

## Why It's Good For The Game

someone posted this on reddit (https://www.reddit.com/r/SS13/comments/i1l3nl/mousehat/):
image

## Changelog
:cl: ATHATH
fix: You can no longer store 5 million live mice inside of a chef's hat.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
addresses the largest issue with our codebase, speedmerge.

**Comment by ATH1909** [comment]
https://www.youtube.com/watch?v=32ubb1N9hwA

**Comment by tiramisuapimancer** [comment]
feature

File: code/datums/components/storage/concrete/pockets.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	if(istype(I,/obj/item/clothing/head/mob_holder))
 		var/obj/item/clothing/head/mob_holder/mausholder = I
 		if(locate(/mob/living/simple_animal/mouse) in mausholder.contents)
-			return TRUE
+			return .
```
Comment (Rohesie) [inline code/datums/components/storage/concrete/pockets.dm:53]:
```suggestion
			return
```

Comment (ShizCalev) [reply code/datums/components/storage/concrete/pockets.dm:53]: shouldn't this be false?

File: code/datums/components/storage/concrete/pockets.dm [hunk 2]
```diff
@@ -50,7 +50,7 @@
...
 		return FALSE
```
Comment (ShizCalev) [inline code/datums/components/storage/concrete/pockets.dm:54]: and this should just be a return?

File: code/datums/components/storage/concrete/pockets.dm (hunk 1 above)
Comment (ATH1909) [reply code/datums/components/storage/concrete/pockets.dm:53]: Does "return" with no "." after it return the value of ".", or just FALSE?

File: code/datums/components/storage/concrete/pockets.dm (hunk 2 above)
Comment (ATH1909) [reply code/datums/components/storage/concrete/pockets.dm:54]: Does "return" with no "." after it return the value of ".", or just FALSE?

File: code/datums/components/storage/concrete/pockets.dm (hunk 1 above)
Comment (Rohesie) [reply code/datums/components/storage/concrete/pockets.dm:53]: It returns the value of `.`
Check the reference:
http://www.byond.com/docs/ref/index.html#/proc/var/.
http://www.byond.com/docs/ref/index.html#/proc/return

File: code/datums/components/storage/concrete/pockets.dm (hunk 2 above)
Comment (ATH1909) [reply code/datums/components/storage/concrete/pockets.dm:54]: if this returns just ., then mouse-less mob_holders (like, uh, drones and stuff, I guess) will be able to fit inside of the hat

**Comment by dondonpow** [comment]
/tg/ just keeps on removing features smh

**Pull request merged**